### DISCUSSION TITLE
Non-existant/incorrect paths show no errors

### DISCUSSION THREAD

**Discussion by douglasg14b** [vuejs/language-tools#3768] [Q&A] [answered]
When I have an invalid component import, no errors are shown.

I'm using the TypeScript Vue Plugin.

For example:

```vue
<script setup lang="ts">
import Logo from '../LogoTypo.vue';
</script>
```

I'm importing from a file that does not exist, yet there are no errors, or warnings. nd the import is typed as `(alias) const Logo: DefineComponent<{}, {}, any>`

![image](https://github.com/vuejs/language-tools/assets/1400380/251b1024-252d-45cb-8949-98806dc7271e)

How can I have errors for invalid imports with Volar?

**Comment by so1ve** [comment, marked as answer]
Remove the `d.ts` file that contains `declare module "*.vue"`

**Reply by douglasg14b** [reply]
Seems like Vuetify project scaffolding is bad.
